src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -170,6 +170,7 @@ export default {
 
 				this.$refs[key][0].addToBasket(amount);
 			}
+			this.$kvTrackEvent('event-tracking', 'add-all-to-basket', 'relending-lending-home');
```

Comment:
The action `add-all-to-basket` is not on our list of web event actions. I would suggest: 
'loan-card', 'add-to-basket', 'relending-lending-home-add-all'

But I don't know what this looks like, a screenshot would help, see: https://docs.google.com/spreadsheets/d/1pbxxYDfeOkrGhFi8HWl5K-vCPXaqO3bXS1Io3E_SDKc/edit?pli=1#gid=1801894844
for analytics actions and events

Reply:
Right! Lauren asked to be like that, but I think your suggestion should be useful for them. I'll ping her about it.

Thanks @eddieferrer !

Reply:
@eddieferrer only thing is that is not a loan card, is button in the top of the section adding multiple loans to the basket... agree having `event-tracking` or what do you suggest for this?

Reply:
<img width="1177" alt="Screenshot 2023-05-04 at 13 57 26" src="https://user-images.githubusercontent.com/94026278/236315745-f68a60c0-6a68-4964-aa37-da04df15362d.png">

Reply:
The action does trigger the loan card action, so using the loan-card signifier should be appropriate enough.